packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
```

Comment:
It would be great to start the README.md to explain what the package is for and how it should be used, rather than how to test it.